### PR TITLE
feat: support pvc expansion

### LINKED ISSUE #194: Support PVC expansion

### PR DESCRIPTION
**Pull request by kqzh**
## What type of PR is this?
- [ ] bug
- [x] feature
- [ ] enhancement

#### Issue(s) number: 
- fix #194

### CODE REVIEW

File: pkg/controller/component/helper.go [hunk 1]
```diff
@@ -504,3 +504,36 @@ func updatePod(clientSet kube.ClientSet, newPod, oldPod *corev1.Pod) error {
...
+		for i := 0; i < replicas; i++ {
+			pvcName := fmt.Sprintf("%s-%s-%d", volumeClaim.Name, component.GetName(), i)
+			oldPVC, err := pvcClient.GetPVC(component.GetNamespace(), pvcName)
+			if err != nil {
+				if !apierrors.IsNotFound(err) {
+					return err
+				}
+			}
+			if oldPVC == nil {
+				return nil
```
Comment (veezhang) [inline pkg/controller/component/helper.go:527]: If the first one is nil, then other volumeClaim and replicas do not need to be processed?

**Comment by veezhang** [comment]
How about support automatic expansion?

File: pkg/controller/component/helper.go [hunk 2]
```diff
@@ -504,3 +504,36 @@ func updatePod(clientSet kube.ClientSet, newPod, oldPod *corev1.Pod) error {
 func isPending(pod *corev1.Pod) bool {
 	return pod.Status.Phase == corev1.PodPending
 }
+
+// 1. new pvc is not nil, old pvc is not nil, update storage
+func syncPVC(
```
Comment (veezhang) [inline pkg/controller/component/helper.go:509]: Is it necessary to check whether pvc supports expansion?

**Review by MegaByte875 — APPROVED** [review]
LGTM

**Comment by wey-gu** [comment]
So very excited about this feature!!!

**Pull request merged**